docker/Dockerfile

```
@@ -85,13 +85,13 @@ RUN set -eux; \
 FROM node:20.18.0-alpine
 
 # Install required system packages
-RUN apk add --no-cache python3 jq git
+RUN apk add --no-cache python3 jq git openjdk17-jre
```

Hiero Consensus Node requires Java 21 LTS. 

```suggestion
RUN apk add --no-cache python3 jq git openjdk21-jre
```

---

@belloibrahv , see Nathan's comment above